src/screens/auth/StartScreen.tsx

```diff
@@ -33,15 +26,15 @@ export const StartScreen = ({ route, navigation }: any) => {
           <H1Heading>{"Hello! Let's\nget started."}</H1Heading>
 
           <ButtonContainer>
-            <ButtonWhite onPress={toLoginScreen}>
+            <ButtonWhite onPress={() => navigation.navigate('Login')}>
               <DarkGrayText>
                 <H4_Card_Nav_Tab>Login</H4_Card_Nav_Tab>
               </DarkGrayText>
             </ButtonWhite>
           </ButtonContainer>
 
           <ButtonContainer>
-            <ButtonMagenta onPress={toSignupScreen}>
+            <ButtonMagenta>
```

Comment:
Nit (only applicable if we merge this before signup gets merged): add a TODO here to integrate navigation